### ISSUE TITLE
HF0.2 cli_wallet: get_account_history does not work 

### ISSUE THREAD

**Issue by gropox** [GolosChain/golos#306]
get_account_history does not work if there are new operations in transaction history. api call get_account_history works fine

```
unlocked >>> get_account_history cyberfounder -1 100
get_account_history cyberfounder -1 100
{"level":"warn","file":"websocket_api.cpp","line":86,"method":"on_message_impl","hostname":"","thread_name":"th_a","timestamp":"2017-11-26T16:22:42","context":"","format":"message: ${message} ","data":{"message":"{\"id\":58,\"result\":[[0,{\"trx_id\":\"0000000000000000000000000000000000000000\",\"block\":1,\"trx_in_block\":0,\"op_in_trx\":0,\"virtual_op\":0,\"timestamp\":\"2017-11-26T16:20:09\",\"op\":[\"custom\",{\"required_auths\":[\"cyberfounder\"],\"id\":0,\"data\":\"546573746e65743a2048617264666f726b206170706c696564\"}]}],[1,{\"trx_id\":\"6477525b45aadae5c93e6225b6df7b40e93414ab\",\"block\":51,\"trx_in_block\":0,\"op_in_trx\":0,\"virtual_op\":0,\"timestamp\":\"2017-11-26T16:22:36\",\"op\":[\"asset_create\",{\"issuer\":\"cyberfounder\",\"asset_name\":\"GBT\",\"precision\":3,\"common_options\":{\"max_supply\":\"1000000000000000\",\"market_fee_percent\":0,\"max_market_fee\":\"1000000000000000\",\"issuer_permissions\":79,\"flags\":0,\"core_exchange_rate\":{\"base\":\"1.000 GBT\",\"quote\":\"1.000 GOLOS\"},\"whitelist_authorities\":[],\"blacklist_authorities\":[],\"whitelist_markets\":[],\"blacklist_markets\":[],\"description\":\"Business Asset\",\"extensions\":[]},\"is_prediction_market\":false,\"extensions\":[]}]}]]}"}}
10 basic_exception: tion
itr != name_map.end(): Invalid operation name: asset_create_operation<0, 16, 0>
    {"n":"asset_create_operation<0, 16, 0>"}
    th_a  operations.cpp:119 from_variant

```

**Comment by Nemo1369** [comment]
Thanks for the issue suggestion. Seems like it is an account_history_plugin operations unpack bug.

**Comment by Nemo1369** [comment]
Fixed with 4f6ab212e3
